site/source/docs/api_reference/emscripten.h.rst

```diff
@@ -931,6 +931,15 @@ Functions
 	:returns: A pointer to the preloaded image or NULL.
 	:rtype: char*
 
+.. c:function:: char *emscripten_print_double(double x, char *to)
```

Comment:
I think we should always favor safe forms, and require here that as a third parameter, the number of bytes that this function can write (including null terminator) is passed to emscripten_print_double. That way accidental buffer overflows are reduced.